### ISSUE TITLE
Support yaml config and use new .jsbeautifyrc.json config style

### ISSUE THREAD

**Issue by joelharkes** [beautify-web/js-beautify#1181]
# Description

eslint and stylelint both moved their config to the style where the file must be suffixed with the format of the data: .json and .yml and depcrecated the .eslintrc format (without suffix).

It would be nice for this library to follow this convention: http://eslint.org/docs/user-guide/configuring#configuration-file-formats

**Comment by Kristinita** [comment]
+1. [**Similar issue**](https://github.com/DavidAnson/markdownlint/issues/118).

+ [**JSON as configuration files: please don't**](https://arp242.net/weblog/json_as_configuration_files-_please_dont)

Thanks.

**Comment by bitwiseman** [comment]
@Kristinita 
I agree with the intent.  Thanks for the added link.  
https://github.com/beautify-web/js-beautify#contributors-needed

**Comment by Kristinita** [comment]
@joelharkes , is this issue solved?

Thanks.

**Comment by bitwiseman** [comment]
@Kristinita Nope, they just decided to close the issue for some reason.

**Comment by joelharkes** [comment]
Yes decided to close my long running issues I'm no longer involved with.

**Comment by Kristinita** [comment]
**Type: hint**

### Cosmiconfig

[**Cosmiconfig**](https://github.com/davidtheclark/cosmiconfig) is the popular and not very hard in use configuration loader.

1. [**Usage**](https://github.com/davidtheclark/cosmiconfig#usage)
1. Example — [**stylelint**](https://stylelint.io/user-guide/configuration#loading-the-configuration-object)

Thanks.
